cranelift/codegen/src/isa/aarch64/lower/isle.rs

```diff
@@ -152,14 +152,17 @@ where
                         let imm =
                             MoveWideConst::maybe_with_shift(((!imm16) & 0xffff) as u16, i * 16)
                                 .unwrap();
-                        self.emitted_insts.push(MInst::MovN { rd, imm, size });
+                        self.emitted_insts
+                            .push((MInst::MovN { rd, imm, size }, false));
                     } else {
                         let imm = MoveWideConst::maybe_with_shift(imm16 as u16, i * 16).unwrap();
-                        self.emitted_insts.push(MInst::MovZ { rd, imm, size });
+                        self.emitted_insts
+                            .push((MInst::MovZ { rd, imm, size }, false));
                     }
                 } else {
                     let imm = MoveWideConst::maybe_with_shift(imm16 as u16, i * 16).unwrap();
-                    self.emitted_insts.push(MInst::MovK { rd, imm, size });
+                    self.emitted_insts
+                        .push((MInst::MovK { rd, imm, size }, false));
```

Comment:
Could we use `self.emit(..)` here to hide the bool tuple entry, which is noise/distracting/raises irrelevant questions for new code readers who stumble upon this?

Reply:
Ah I see this already merged, I'll send a follow up PR.